java/build.sh

```diff
@@ -3,12 +3,24 @@ GROUP_ID="com.nvidia.cuvs"
 SO_FILE_PATH="./internal"
 
 if [ -z "$CMAKE_PREFIX_PATH" ]; then
-    export CMAKE_PREFIX_PATH=`pwd`/../cpp/build
+  export CMAKE_PREFIX_PATH=`pwd`/../cpp/build
```

Comment:
```suggestion
  export CMAKE_PREFIX_PATH="$(pwd)/../cpp/build"
```

Can we please use the `$()` form instead? When `shellcheck` linting eventually starts to cover this file, it will complain about the use of backticks, for the reasons mentioned in https://www.shellcheck.net/wiki/SC2006